client/components/Review.js

```diff
@@ -0,0 +1,19 @@
+import React from  'react';
+
+export const Review = ({ review }) => {
+
+  const stars = (len) => {
+    const starArray = []
+    for(let i = 0; i < len; i++) starArray.push(<img key={i} src='/img/star.png'/>)
```

Comment:
Something's not working here with the image of the star because it's not loading. Did you push the image file to the branch? I can't find it.

Reply:
Dang. I took it out bc it wasn't used in the review form but I forgot that it was here as well. I'll update this part to use the star emoji like the review form.

Reply:
done